firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackSender.java

```diff
@@ -32,11 +34,18 @@ static FeedbackSender getInstance() {
     return FirebaseApp.getInstance().get(FeedbackSender.class);
   }
 
-  /** Send feedback text and screenshot to the Tester API for the given release. */
-  Task<Void> sendFeedback(String releaseName, String feedbackText, Bitmap screenshot) {
+  /** Send feedback text and optionally a screenshot to the Tester API for the given release. */
+  Task<Void> sendFeedback(String releaseName, String feedbackText, @Nullable Bitmap screenshot) {
     return testerApiClient
         .createFeedback(releaseName, feedbackText)
-        .onSuccessTask(feedbackName -> testerApiClient.attachScreenshot(feedbackName, screenshot))
+        .onSuccessTask(feedbackName -> attachScreenshot(feedbackName, screenshot))
         .onSuccessTask(testerApiClient::commitFeedback);
   }
+
+  private Task<String> attachScreenshot(String feedbackName, @Nullable Bitmap screenshot) {
+    if (screenshot == null) {
+      return Tasks.forResult(feedbackName);
+    }
+    return testerApiClient.attachScreenshot(feedbackName, screenshot);
```

Comment:
Maybe this should now get the filename of the screenshot instead? If I understand correctly we already have a compressed file - so we should be able to simply write that to the network instead of compressing the bitmap (again)